### PR TITLE
support large payload in zk put API

### PR DESCRIPTION
**Pull request by galalen**
## Description
support large payload in ZK put API, solve [issue 7305](https://github.com/apache/pinot/issues/7305)

## Upgrade Notes
Does this PR prevent a zero down-time upgrade? (Assume upgrade order: Controller, Broker, Server, Minion)
* [ ] Yes (Please label as **<code>backward-incompat</code>**, and complete the section below on Release Notes)

Does this PR fix a zero-downtime upgrade introduced earlier?
* [x] Yes (Please label this as **<code>backward-incompat</code>**, and complete the section below on Release Notes)

Does this PR otherwise need attention when creating release notes? Things to consider:
- New configuration options
- Deprecation of configurations
- Signature changes to public methods/interfaces
- New plugins added or old plugins removed
* [ ] Yes (Please label this PR as **<code>release-notes</code>** and complete the section on Release Notes)

### CODE REVIEW

**Comment by xiangfu0** [comment]
Thanks for adding this!

Can you add a test for this?

**Comment by galalen** [comment]
@Jackie-Jiang can you check the PR now, I'll be updating tests coverage

File: pinot-controller/src/test/java/org/apache/pinot/controller/api/ZookeeperResourceTest.java [hunk 1]
```diff
@@ -0,0 +1,101 @@
+package org.apache.pinot.controller.api;
```
Comment (Jackie-Jiang) [inline pinot-controller/src/test/java/org/apache/pinot/controller/api/ZookeeperResourceTest.java:1]: Missing license header

File: pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/ZookeeperResource.java [hunk 2]
```diff
@@ -133,13 +136,23 @@ public SuccessResponse delete(
...
       @DefaultValue("-1") String expectedVersion,
-      @ApiParam(value = "accessOption", required = true, defaultValue = "1") @QueryParam("accessOption")
-      @DefaultValue("1") String accessOption) {
+      @ApiParam(value = "accessOption", defaultValue = "1") @QueryParam("accessOption")
+      @DefaultValue("1") String accessOption,
+      @Nullable Payload payload
+  ) {
+
+    if (payload != null) {
+      path = payload.getPath();
```
Comment (Jackie-Jiang) [inline pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/ZookeeperResource.java:150]: We should only override the value when it is provided in the payload

**Review by Jackie-Jiang — APPROVED** [review]
LGTM otherwise

File: pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/ZookeeperResource.java (hunk 2 above)
Comment (galalen) [reply pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/ZookeeperResource.java:150]: @Jackie-Jiang updated the requested changes

**Pull request merged**